load/InputFileset.py

```diff
@@ -64,9 +67,16 @@ def filesetCommandLineParser(config, s3Client, lptsReader):
 		filesetPaths = sys.argv[3:]
 
 		for filesetPath in filesetPaths:
+			stocknumbers = PreValidate.getStockHumbersFromDirectory(filesetPath, location.replace("s3://", ""))
+			hasStocknumbersFile = True if len(stocknumbers) > 0 else False
 			filesetPath = filesetPath[:-1] if filesetPath.endswith("/") else filesetPath
-			filesetId = filesetPath.split("/")[-1]
-			filesetId = filesetId[:7] + "_" + filesetId[8:] + "-usx" if filesetId[8:10] == "ET" else filesetId
+
+			directory = filesetPath.split("/")[-1]
+			filesetId = directory
+
+			if hasStocknumbersFile == True:
```

Comment:
shouldn't this be a check for hasStocknumbersFile = False? we only parse the filesetid from the directory name if stocknumbers.txt doesn't exist

Reply:
@bradflood, you are right. I have changed it.